pkgs/applications/misc/blender/default.nix

```diff
@@ -5,7 +5,7 @@
 , openvdb, libXxf86vm, tbb, alembic
 , zlib, zstd, fftw, opensubdiv, freetype, jemalloc, ocl-icd, addOpenGLRunpath
 , jackaudioSupport ? false, libjack2
-, cudaSupport ? config.cudaSupport or false, cudatoolkit_11
+, cudaSupport ? config.cudaSupport or false, cudaPackages_11 ? {}
```

Comment:
Does it have to be `cudaPackages_11` over just `cudaPackages`?

Reply:
No, I just did not want to change it, thinking there might have been a reason for it.